docs/vision/lifetime.md

```diff
@@ -0,0 +1,80 @@
+# Lifetime
+
+DropIt is unique in the Polkadot ecosystem in that it is designed to be **ephemeral**.
+
+Whereas most parachains take careful consideration to ensure their chain can scale and persist over time, DropIt will be designed to grow fast and without concern for the far future.
+
+To understand what this means, you need to understand a few topics.
+
+## State Bloat
+
+[State bloat](https://www.techopedia.com/definition/blockchain-bloat) is one of the key long term scaling issues of blockchain systems. As the chain grows, accessing items in the database become more complex due to the construction of the Merkle trie. Also running a full node for the chain becomes more expensive as more storage space is needed to keep the current state of the chain.
+
+Chain's like Bitcoin are plagued with ["dust accounts"](https://bitcoinwiki.org/wiki/cryptocurrency-dust) whose balance is lower than the cost of transaction fees needed to move those balances. Thus these accounts will likely congest the network's Merkle trie forever, and result in a decrease of performance and an increase of costs when running Bitcoin nodes.
```

Comment:
Bitcoin uses UTXOs so there's no merkle trie of accounts, maybe just use Ethereum as an example with both dust accounts and bloat from contract state.